app/scripts/background.js

```diff
@@ -367,7 +386,7 @@ function setupController(initState, initLangCode) {
       // communication with popup
       controller.isClientOpen = true;
       controller.setupTrustedCommunication(portStream, remotePort.sender);
-
+      remotePort.postMessage({ name: 'CONNECTION_READY' });
```

Comment:
This message is causing warning:
![Screenshot 2022-03-22 at 6 22 58 AM](https://user-images.githubusercontent.com/2182307/159386364-1f80fc5d-3d99-44bd-9c09-e95253061563.png)